frontend/src/metabase/query_builder/components/view/View/NotebookContainer/NotebookContainer.module.css

```diff
@@ -0,0 +1,36 @@
+:root {
+  --notebook-width: 40rem;
+  --sidebar-width: 428px;
```

Comment:
Another very specific request from the Figma design. I don't think `26.75rem` would look great here, but happy to change it from `px` to `rem` if needed.